lib/grape/formatters/active_model_serializers.rb

```diff
@@ -0,0 +1,16 @@
+# A grape response formatter that can be used as 'formatter :json, Grape::Formatters::ActiveModelSerializers'
+#
+# Serializer options can be passed as a hash from your grape endpoint using env[:active_model_serializer_options],
+# or better yet user the render helper in Grape::Helpers::ActiveModelSerializers
+module Grape
+  module Formatters
+    module ActiveModelSerializers
+      def self.call(resource, env)
+        return resource.to_json unless resource.is_a?(Array) || resource.respond_to?(:model_name) || resource.class.respond_to?(:model_name)
```

Comment:
@bf4 that was put there by the previous developer, I have no idea why it was there, but assumed it was OK due to it being in the previous PR